pid/pid.go

```diff
@@ -7,7 +7,7 @@ import (
 	"syscall"
 )
 
-var FileDir = "/var/run/lukso"
+var FileDir = "."
```

Comment:
Note for reviewers - this is for easy node running, making a `/var/run/lukso` needs to be done manually, if anyone wants to run a lukso node right now then they don't need to create any directories themselves - if we could figure out how exactly PID files should be stored it would be great, but for now I'll just store them there

Reply:
Can't you mkdir for user?

Reply:
Yeah, I can do that actually